azure_img_utils/azure_image.py

```diff
@@ -603,7 +625,7 @@ def publish_offer(
             }
         ]
 
```

Comment:
We can delete the headers variable in line 592 now the headers are got in `submit_request`

Reply:
I don't see this line in the code. There are methods where headers are still retrieved because they are used in different requests such as `publish_offer`.

Reply:
Don't see this issue anymore so will mark resolved. Some of the methods do get headers still but they are used to get durable id. Please re-open if I'm missing something.